### PR TITLE
Add brain damage line

### PR DESCRIPTION
**Pull request by ghost**
based on this wonderful post in the last thread

### CODE REVIEW

**Comment by CptWad** [comment]
t2fhamo

**Comment by ghost** [comment]
Good job

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 1]
```diff
@@ -60,6 +60,7 @@
 						"SARAH HALE DID IT!!!", \
 						"Don't tell Chase", \
 						"not so tough now huh", \
+						"If you faggots don't like the PR's why don't you just code your own?", \
```
Comment (Intigracy) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: >correct punctuation
>correct spelling
>correct grammar
>correct capitalization

how is this a brain damage line again?

Comment (ghost) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: the spirit of the message

let me make it a little more retarded

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm [hunk 2]
```diff
@@ -60,6 +60,7 @@
 						"SARAH HALE DID IT!!!", \
 						"Don't tell Chase", \
 						"not so tough now huh", \
+						"IF YOU FAGGETS DONT LIKE THE PEEARES WHY DONT YOU CODE YER OWN?!", \
```
Comment (SonixApache) [inline code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: pee arrs?

Comment (Shadowmech88) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: PRs.

Comment (Kurfursten) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: Pull requests

**Comment by MadmanMartian** [comment]
seems a little ooc in ic

**Comment by Exxion** [comment]
I agree. I don't think this one fits very well.

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 2 above)
Comment (SonixApache) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: i mean it literally
pee arrs or PEEARES
i feel the former is better

**Comment by SonixApache** [comment]
have you seen the other ones

did you know that when you try to attach a pipe that isn't connected to it the game tells you the game can't let you do that

that said, what about just `WHEN YOU CODE IT`

File: code/modules/mob/living/carbon/human/life/handle_disabilities.dm (hunk 2 above)
Comment (Exxion) [reply code/modules/mob/living/carbon/human/life/handle_disabilities.dm]: pears

**Comment by Blithering** [comment]
What about something based on the last line? "Your lazy, dumb. Janitor and silly little service roles. I've been RD for six years. Don't talk to me about [slimes/borgs/anything]"

**Comment by ghost** [comment]
the whole thing is worthy of multiple brain damage lines
@Exxion 
@MadmanMartian 
we have multiple brain damage lines that are similar, this new one has a similar theme

` >my face`

>  lol2cat

> lifelike texture ;_;"

I don't see why this one doesn't fit right in

**Comment by Exxion** [comment]
It's not really similar to any of those

**Comment by N3X15** [comment]
A few options:
> IF YOU DONT LIKE THE CYBORGS OR SLIMES WHY DONT YU O JUST MAKE YORE OWN

> YOU AR JUS LAZY AND DUMB JAMITORS AND SERVICE ROLLS

> I WUS ARRDEE FOR TWENNYSEVEN YEARS

> DONT TALK TO ME ABOUT BALANCE!!!!

**Comment by MadmanMartian** [comment]
>DONT TALK TO ME ABOUT BALANCE

Can we have them fall over when saying this?

**Pull request merged**